iphone/Maps/Classes/CarPlay/Template Builders/SettingsTemplateBuilder.swift

```diff
@@ -37,8 +36,7 @@ final class SettingsTemplateBuilder {
   private class func createUnpavedButton(options: RoutingOptions) -> CPGridButton {
     var unpavedIconName = "ic_carplay_unpaved"
     if options.avoidDirty { unpavedIconName += "_active" }
-    let unpavedButton = CPGridButton(titleVariants: [L("avoid_unpaved_carplay_1"),
-                                                     L("avoid_unpaved_carplay_2")],
+    let unpavedButton = CPGridButton(titleVariants: [L("avoid_unpaved")],
```

Comment:
@AndrewShkrob CarPlay supports several titles for different sizes/lengths. Is it also supported in AA?